docs/src/main/asciidoc/config-your-quarkus-applications-using-a-yaml-file_howto.adoc

```diff
@@ -0,0 +1,407 @@
+////
```

Comment:
@quarkusio/documentation do we really want the URL to be that long? Just wondering.

Reply:
also, it should be -howto, not _howto.

Reply:
Also, do we want to break the old link?

Reply:
Shortened it to `config-yaml-howto.adoc`. The new Quarkus docs landing page requires the `-howto` extension.